sys/net/vaina/vaina.c

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (C) 2020 Locha Inc
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_vaina
+ * @{
+ *
+ * @file
+ * @brief       VAINA - Versatile Address Interface | Network Administration
+ *
+ * @author      Jean Pierre Dudey <jeandudey@hotmail.com>
```

Comment:
```suggestion
 * @author      Locha Mesh developers <developers@locha.io>
```

Reply:
Locha copyright is at the top, don't you think it makes sense to leave code authors in the docs so everyone knows who to ask in case help is needed?

Reply:
all the code follow a similar line, PE, 
```
 * @file main.cpp
 * @author Locha Mesh Developers (contact@locha.io)
 * @brief
...
```
AUTHORS or CONTRIBUTORS file can be similar to RIOT-OS:
[section] 
- [name_author_1], [email_author_1]
- [name_author_2], [email_author_2]
.... and so

Reply:
afaik RIOT maintains only the codeowners files only for reviews.

Reply:
yes, for it [section] and authors

Reply:
in compliance with all existing code

Reply:
file header in compliance with all existing code